usr/bin/ublue-user-setup

```diff
@@ -17,6 +17,13 @@ IMAGE_NAME=$(jq -r '."image-name"' < $IMAGE_INFO)
 # Enable NTP
 timedatectl set-ntp true
 
+if [[ $IMAGE_NAME = "bluefin-dx" ]]; then
```

Comment:
Instead of `=`, use `=~` for comparison. Different variants will have different names. I.E. `$IMAGE_NAME` for an NVIDIA system is `bluefin-dx-nvidia`